### PR TITLE
[ML-918] Fixed the data prepration

### PR DESCRIPTION
**Pull request by anuj-sadani-infrrd**
### Jira
[ML-918](https://infrrd.atlassian.net/browse/ML-918)

### Description
Describe your changes, explain what you've done, how and why.
- [x] Fixed the IOB formation creation  

### Creator checklist:
- [x] The code follows the style guidelines of this project
- [x] Run on local machine and checked the fix or the feature functionality
- [ ] Compatibility check: tested my code together with the other services
- [ ] Documentation is updated (if necessary)

#### How to test
- Required database-manager changes (https://github.com/infer-ai/feedback-database-manager/pull/177)
- `git checkout ML-918` for feedback-database-manager repo
- `git pull origin ML-918` (stash any changes before running this)
- `pip install -e /home/anuj/gitrepo/feedback-database-manager` install the package (update the path to you local dir)

### Reviewer checklist:
- [ ] I thoroughly reviewed the modified code
- [ ] I pulled the branch to my local and, if necessary, ran migrations and pip install
- [ ] Manual tests (Optional)

**As a reviewer**, if you see something not checked, please do not `+1` or `LGTM` the PR.

### CODE REVIEW

File: blueprints/utils/util.py [hunk 1]
```diff
@@ -184,6 +184,7 @@ def create_entity_mapping_file(entities, location):
 
 
 def update_status_and_message_for_cluster_trainer_detail(message, status, cluster_trainer_detail):
+    return
```
Comment (abhishek-biswas-infrrd) [inline blueprints/utils/util.py:187]: are we returning something using return keyword here ?

Comment (anuj-sadani-infrrd) [reply blueprints/utils/util.py:187]: I pushed the testing code :face_exhaling: 
Let me fix this.

Comment (anuj-sadani-infrrd) [reply blueprints/utils/util.py]: corrected.

**Review by abhishek-biswas-infrrd — APPROVED** [review]
LGTM

**Pull request merged**